src/components/navigation/TheUserMenu.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <v-menu v-if="!mobile" offset-y>
+  <v-btn
+    dark
+    icon
+    style="margin-right:2px"
+    v-if="!mobile && !isAuth"
+    @click="doOpen"
```

Comment:
handler methods should be named accordingly.
"blahBlahOnClickHandler"
"blahBlahOnChangeHandler"
Where the BlahBlah describes what that button is doing such as
"userProfileMenuOnClickHandler"